src/components/image-manager/advanced-pull.js

```diff
@@ -22,6 +22,7 @@ const defaultImagesList = [
   'openchemistry/nwchem:6.6',
   'openchemistry/psi4:1.2.1',
   'openchemistry/torchani:1.2',
+  'openchemistry/qiskit:1.0'
```

Comment:
It looks like for the other containers we are using the version from the package it encapsulated, what version of qiskit are we using, we should probably use that as the tag.